resources/lang/lao.ini

```diff
@@ -0,0 +1,112 @@
+error.nosession = "ບໍ່ມີເຊສຊັ່ນສ້າງຂຶ້ນ - ອາດເປັນເພາະບໍ່ມີສິດນຳໃຊ້ {%0}"
+error.noclipboard = "ບໍ່ມີຄລິບບອດ - ສ້າງຄລິບບອດກ່ອນ"
+warning.differentworld = "[ຄຳເຕືອນ] ທ່ານແກ້ໄຂເລເວລທ່ານບໍ່ໄດ້ຢູ່!"
+; commands
+command.info.title = "ຂໍ້ມູນ"
+command.limit.current = "ຈຳນວນຈຳກັດປະຈຸບັນ: {%0}"
+command.limit.set = "ຈຳນວນຈຳກັດໃນການແກ້ໄຂບລັອກຖືກຕັ້ງເປັນ {%0}"
+command.setrange.current = "ໄລຍະປັດຈຸບັນ: {%0}"
+command.setrange.set = "ໄລຍະອຸປະກອນຖືກຕັ້ງເປັນ {%0}"
+command.biomeinfo.attarget = "Biome ທີ່ເປົ້າໝາຍ"
+command.biomeinfo.atposition = "Biome ທີ່ຕຳແໜ່ງ"
+command.biomeinfo.result = "ພົບ {%0} biomes ໃນການເລຶອກ"
+command.biomeinfo.result.line = "ໄອດີ: {%0} ຊື່: {%1}"
+command.biomelist.title = "ລາຍຈຳນວນຂອງ Biome"
+command.biomelist.result.line = "ໄອດີ: {%0} ຊື່: {%1}"
+command.brushname.set = "ຊື່ຖືກປ່ຽນເປັນ \"{%0}\""
+command.clearclipboard.cleared = "ຄລິບບອດຖືກລົບລ້າງ"
+command.flip.try = "ພະຍາຍາມທີ່ຈະພິກຄລິບບອດໂດຍ {%0}"
+command.flip.success = "ຄລິບບອດຖືກພິກຮຽບຮ້ອຍ"
+command.rotate.try = "ພະຍາຍາມທີ່ຈະພິກຄລິບບອດ {%0} ອົງສາ"
+command.rotate.success = "ຄລິບບອດຖືກໝຸນຮຽບຮ້ອຍແລ້ວ"
+command.history.cleared = "ປະຫວັດຖືກລົບລ້າງແລ້ວ"
+command.listchunks.found = "ພົບ {%0} chunks ໃນການເລືອກ"
+command.size = "ຂະໜາດການເລືອກ"
+; selection
+selection.pos1.set = "ຕຳແໜ່ງທີ່ 1 ຕັ້ງເປັນ X: {%0} Y: {%1} Z: {%2}"
+selection.pos2.set = "ຕຳແໜ່ງທີ່ 2 ຕັ້ງເປັນ X: {%0} Y: {%1} Z: {%2}"
+; session
+session.undo.none = "ຍ້ອນກັບບໍ່ໄດ້"
+session.undo.left = "ທ່ານມີຄຳສັ່ງຍ້ອນກັບເຫຼືອ {%0} ເທື່ອ"
+session.redo.none = "ບໍ່ມີຫຍັງໃໝ່ໃຫ້ເຮັດ"
+session.redo.left = "ທ່ານມີຄຳສັ່ງໃໝ່ໃຫ້ເຮັດ {%0}"
+session.brush.added = "ເພີ່ມ {%0} ເຂົ້າເຊສຊັ້ນ"
+session.brush.deleted = "ລົບ {%0} (UUID {%1})"
+session.brush.removed = "ນຳອອກ {%0} (UUID {%1})"
+session.language.set = "ປ່ຽນພາສາເປັນ {%0} ຮຽບຮ້ອຍ"
+session.language.notfound = "ບໍ່ພົບພາສາ {%0} ປັບເປັນຄ່າເລີ່ມຕົ້ນ"
+; task
+task.copy.success = "ຄັດລອກສຳເລັດ, ໃຊ້ເວລາ {%0}, ຄັດລອກ {%1} ບລັອກຈາກທັງໝົດ {%2}"
+task.count.success = "ວິເຄາະສຳເລັດ, ໃຊ້ເວລາ {%0}"
+task.count.result = "ພົບ {%0} ບລັອກທັງໝົດ {%1} ບລັອກ"
+task.fill.success = "ເຕີມເຕັມສຳເລັດ, ໃຊ້ເວລາ {%0}, ປ່ຽນ {%1} ບລັອກຈາກທັງໝົດ {%2}"
+task.replace.success = "ວາງທັບສຳເລັດ ໃຊ້ເວລາ {%0}, ປ່ຽນ {%1} ບລັອກຈາກທັງໝົດ {%2}"
+task.revert.undo.success = "ຍ້ອນກັບສຳເລັດ, ໃຊ້ເວລາ {%0}, ປ່ຽນ {%1} ບລັອກຈາກທັງໝົດ {%2}"
+task.revert.redo.success = "ເຮັດໃໝ່ສຳເລັດ, ໃຊ້ເວລາ {%0}, ປ່ຽນ {%1} ບລັອກຈາກທັງໝົດ {%2}"
+; flags
+flags.keepexistingblocks = "ເກັບບລັອກທີ່ມີຢູ່ສຳເລັດ"
+flags.keepair = "ເກັບອາກາດ"
+flags.hollow = "ໂພງ"
+flags.hollowclosed = "ໂພງທີ່ມີທາງອອກປິດ"
+flags.natural = "ຕາມທຳມະຊາດ"
+; tools
+; wand tool
+tool.wand = "ຄະທາ"
+tool.wand.lore.1 = "ຄິກຊ້າຍທີ່ບລັອກເພື່ອຕັ້ງຕຳແໜ່ງ 1 ຂອງການເລືອກ"
+tool.wand.lore.2 = "ຄິກຂວາທີ່ບລັອກເພື່ອຕັ້ງຕຳແໜ່ງ 2 ຂອງການເລືອກ"
+tool.wand.lore.3 = "ໃຊ້ຄຳສັ່ງ //togglewand ເພື່ອສະຫຼັບການເຮັດວຽກຂອງຄະທາ"
+tool.wand.disabled = "ຄະທາຖືກປິດໃຊ້ງານ ໃຊ້ຄຳສັ່ງ //togglewand ເພື່ອເປີດໃຊ້ງານ"
+tool.wand.setenabled = "ຄະທາໄດ້ຖືກ{%0}!"
+; debug tool
+tool.debug = "ເຄື່ອງມື Debug"
```

Comment:
Is there really no translation for this? This tool displays information about blocks

Reply:
yes we say debug because we don't have this word in our language